datasets/download/downloading-data.qmd

```diff
@@ -18,36 +18,58 @@ Follow the dialogue to get authenticated and then click on `Download inbox s3cmd
 ### Install the sda-cli tool
 Follow the guidelines [here](../submission/submission-guide.qmd#install-the-sda-cli-tool) to install the sda-cli tool.
 
-### Generate the public and secret key (**ONLY FOR ENCRYPTED DATA**)
+### Generate the public and secret key 
 The initial step involves creating a crypt4gh keypair using the sda-cli:
 
 ```bash
 ./sda-cli createKey <keypair_name>
 ```
 
 where `<keypair_name>` is the base name of the key files. The above command will create two key files named
-`keypair_name.pub.pem` and `keypair_name.sec.pem`. The public key (`pub`) will be sent to the admins
+`keypair_name.pub.pem` and `keypair_name.sec.pem`. The public key (`pub`) will be used alongside with `sda-cli`
 and will be used by the system for encryption of the files before downloading, while the private one
 (`sec`) will be used by the requester for decrypting the files after downloading.
 
 ## Check access
 
 After the user has been granted access to the dataset, the user can check access to the dataset by listing the datasets and their files using the `sda-cli`.
-This can be done by following the instructions in the `List datasets and their files` section of the `sda-cli` documentation [here](https://github.com/NBISweden/sda-cli#list-datasets-and-their-files).
+For listing the datasets that the user has access to in production environment, the user needs to run:
+
+```bash
+./sda-cli list -config s3cmd.conf --datasets --url https://download.bp.nbis.se (--bytes)
```

Comment:
Better to add or point to instructions about what is  `s3cmd.conf` and how it is obtained.

Reply:
There are instructions about how to obtain it at the beginning of the document